src/store/indexeddb-local-backend.js

```diff
@@ -137,6 +137,24 @@ LocalIndexedDBStoreBackend.prototype = {
     },
 
     /**
+     * Having connected, load initial data from the database and prepare for use
+     */
+    init: function() {
```

Comment:
Not really - I wondered about this but ended up just adding one. Have changed it.